packages/swapper/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './types'
```

Comment:
we should avoid star exports because it breaks treeshaking

Reply:
```suggestion
export { SwapErrorRight, SwapErrorType } from './types'
```

Reply:
https://github.com/shapeshift/web/pull/5792/commits/5606b786435563d33f914f5bee2ac251f99a4a7b